pages/dkp/kommander/2.1/release-notes/2.1.1/index.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: layout.pug
+navigationTitle: Kommander 2.1.1 Release Notes
+title: Kommander 2.1.1 Release Notes
+menuWeight: 20
+excerpt: View release-specific information for Kommander 2.1.1
+enterprise: false
+beta: false
+---
+
+**D2iQ&reg; Kommander&reg; version 2.1.0 was released on December 23, 2021.**
+
+[button color="purple" href="https://support.d2iq.com/hc/en-us/articles/4409215222932-Product-Downloads"]Download Kommander[/button]
+
+To get started with Kommander, [download](../../download/) and [install](../../install/) the latest version of Kommander.
+
+<p class="message--note"><strong>NOTE: </strong>You must be a registered user and logged on to the support portal to download this product. New customers must contact their sales representative or <a href="mailto:sales@d2iq.com">sales@d2iq.com</a> before attempting to download or install Konvoy.</p>
+
+## Release summary
+
+This release provides new features and enhancements to improve the user experience, fix reported issues, integrate changes from previous releases, and maintain compatibility and support for other packages used in Kommander.
+
+## New features and capabilities
+
+<!-- Add new items, if any -->
+
+## Known issues
+
+### DKP delete fails in AWS
+
+When attempting to delete a DKP cluster from AWS where you have permanent credentials, The 'dkp delete' command fails with SIGSEGV. As an alternative, manually delete your cluster. (COPS-7109)
+
+### DKP installer fails to follow respective AWS --region or Azure --location flags
+
+The installer flag for AWS `--region` or Azure `--location`are not enforced in the target cluster. We corrected this isssue and the correct flags generate in the cluster.yaml files. (COPS-7101)
+
+### Broken documentation links
+
+Some hyperlinks on the Major Version Upgrade to DKP page were not working. We corrected these issues. (COPS-7099)
```

Comment:
I don't think we've ever documented fixing docs pages before in release notes, seems kind of strange and Meta since the evidence of the problem is now missing...  :-)   I don't feel like this adds anything useful to the release notes:

```suggestion
```

Reply:
@cneth - we were not sure if customers who entered the COPS ticket follow up, so figured no harm to mention. The rationale being you all in Support can use it as a sort of "look, we're listening to you!" sort of thing, if you want/need it. Or we can take it out, no problem,